Packs/FeedTAXII/ReleaseNotes/1_2_9.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### TAXII 2 Feed
+- Updated the Docker image to: *demisto/taxii2:1.0.0.89439*.
+- Updated the feed parameters order and made adjustments to the API Root param title.
+- Updated the feed description to add clarity on using different API Roots.
```

Comment:
Recommend:
Updated the feed description to add clarity to using different API roots.